cmd/podman/system/prune.go

```diff
@@ -73,32 +74,58 @@ Are you sure you want to continue? [y/N] `, volumeString)
 			return nil
 		}
 	}
-	// TODO: support for filters in system prune
-	response, err := registry.ContainerEngine().SystemPrune(context.Background(), pruneOptions)
-	if err != nil {
-		return err
-	}
-	// Print pod prune results
-	fmt.Println("Deleted Pods")
-	err = utils.PrintPodPruneResults(response.PodPruneReport)
-	if err != nil {
-		return err
-	}
-	// Print container prune results
-	fmt.Println("Deleted Containers")
-	err = utils.PrintContainerPruneResults(response.ContainerPruneReport)
-	if err != nil {
-		return err
-	}
-	// Print Volume prune results
-	if pruneOptions.Volume {
-		fmt.Println("Deleted Volumes")
-		err = utils.PrintVolumePruneResults(response.VolumePruneReport)
+
+	const MAX = 50
```

Comment:
It seems we're now working around the fact that `registry.ContainerEngine().SystemPrune(...)` isn't doing it's job correctly.  Having the code scattered between `cmd/podman` and `pkg/domain/...` seems like a recipe for trouble.

Could we move all the logic to `registry.ContainerEngine().SystemPrune(...)`?  I think it should remove _all data_, even if there are more than 50 iterations.  It's not friendly to use.  If I do a `rm -rf *` I don't want to `ls` afterwards to check if things were really removed.